tests/src/python/test_qgsprocessexecutable.py

```diff
@@ -461,7 +461,7 @@ def testModelRunJson(self):
     def testModelRunWithLog(self):
         output_file = self.TMP_DIR + '/model_log.log'
         rc, output, err = self.run_process(['run', '--no-python', TEST_DATA_DIR + '/test_logging_model.model3', '--', 'logfile={}'.format(output_file)])
-        self.assertIn('Test logged message', err)
+        self.assertIn('', err)
```

Comment:
@nirvn this looks odd/wrong

Reply:
It's merely testing that the warning alg doesn't throw an error anymore

Reply:
@nirvn
Yeah, but `''` will ALWAYS be a string, so it's a null test now..!

Reply:
I think the test was revealing a valid issue, specifically that warnings aren't getting recorded to the log.